### PR TITLE
Show message that public access is disabled

### PR DESCRIPTION
**Pull request by sunu**
On the home page, show a message that public access is currently disabled.

### CODE REVIEW

**Comment by sunu** [comment]
Editing the markdown files to add this message feels a bit weird though. May be putting it in settings is a better move?

Also, looks like the access disabled message on the home screen (https://github.com/alephdata/aleph/blob/main/ui/src/screens/HomeScreen/HomeScreen.jsx#L27) is not being used.

**Comment by sunu** [comment]
This message will show up like this fwiw
image

**Comment by tillprochaska** [comment]
> Editing the markdown files to add this message feels a bit weird though. May be putting it in settings is a better move?

Yes, it’s actually planned to start working on something like that soon!

File: aleph/pages/home.en.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: 'Aleph'
 description: 'Search documents, companies and people'
 html: true
 samples: ["Vladimir Putin", "TeliaSonera"]
+warning_title: "Public access temporarily disabled"
+warning_body: "We have disabled public access to prevent Aleph from being overwhelmed by automated anonymous requests."
```
Comment (tillprochaska) [inline aleph/pages/home.en.md:8]: Is this the correct file to edit? I always thought that aleph.occrp.org will use the site-specific contents from in [`/site/aleph.occrp.org/pages/home.en.md`](https://github.com/alephdata/aleph/blob/main/site/aleph.occrp.org/pages/home.en.md).

Comment (Rosencrantz) [reply aleph/pages/home.en.md:8]: I'd add a date here for context, it can be frustrating not knowing when this started

Comment (sunu) [reply aleph/pages/home.en.md]: Ok, I have updated the PR. Thanks for catching the mix up @tillprochaska!

**Review by Rosencrantz — APPROVED** [review]

**Pull request merged**